.eslintrc.json

```diff
@@ -9,10 +9,16 @@
     "sourceType": "module"
   },
   "rules": {
+    "brace-style": [
+      "error",
+      "stroustrup"
+    ],
```

Comment:
I would like to add `{"allowSingleLine": true}` here for cases like those in `change-view.test.js`. The compactness seems OK to me there’s only one statement that is short and simple.

Reply:
added!